submissions/madmaxWMFU/google-popup/index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>G00gle-style Popup</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+
+<body>
+    <div class="container">
+        <div class="simpleNav">
+            <div class="divLinks">
+                <div><a class="link mail" href="#">Gmail</a></div>
+                <div><a class="link images" href="#">Images</a></div>
+            </div>
+            <div class="divApps">
+                <div>
+                    <label class="checker" for="applications" title="Applications"></label>
+                    <input id="applications" class="applications" type="checkbox" name="">
+                    <input id="moreApps" type="checkbox" name="">
```

Comment:
Good practice to add a label to every input for better accessibility.

Reply:
This mistake was resolved. If you can see the last changes afterward, everything is fine there

Reply:
Good. 
I am starting from the beginning not to miss anything.)